src/axom/mir/doxygen_mainpage.md

```diff
@@ -0,0 +1,46 @@
+MIR {#mirtop}
+=============
+
+Axom's [Mir](@ref axom::mir), (M)aterial (I)interface (R)econstruction component
+provides classes that can process Blueprint meshes with mixed materials into new
+Blueprint meshes such that there is a single material per zone. In addition, the
+MIR component provides useful building blocks for developing algorithms using
+Blueprint meshes. There are views that simplify dealing with Conduit data and
+utility algorithms for processing and constructing meshes.
+
+# Design goals {#goals}
+
+This component's algorithms are mainly delivered as classes that are templated on
+an execution space, allowing them to operate on a variety of computing backends.
+The algorithms take Conduit nodes (containing Blueprint data) as input and they
```

Comment:
remove final word.  The sentence should read "...as input and output new Blueprint data..."'

Reply:
Done